text.go

```diff
@@ -528,8 +528,18 @@ func (t *Text) MostCommonFontFace() FontFace {
 	return family.Face(size*ptPerMm, col, style, variant)
 }
 
-// ToPaths makes a path out of the text, with x,y the top-left point of the rectangle that fits the text (ie. y is not the text base)
-func (t *Text) ToPaths() ([]*Path, []color.RGBA) {
+// RenderAsPath renders the text converted to paths (calling r.RenderPath)
+func (t *Text) RenderAsPath(r Renderer, m Matrix) {
+	paths, colors := t.toPaths()
+	for i, path := range paths {
+		style := DefaultStyle
+		style.FillColor = colors[i]
+		r.RenderPath(path, style, m)
+	}
+}
+
+// toPaths makes a path out of the text, with x,y the top-left point of the rectangle that fits the text (ie. y is not the text base)
+func (t *Text) toPaths() ([]*Path, []color.RGBA) {
```

Comment:
Is `toPaths` actually used in another context? We could merge it with `RenderAsPath`, unless you think it would be useful to have `ToPaths` separately?

Reply:
I haven't seen it being used anywhere else (that's why it is now unexported).

I would rather let it as-is (in 2 methods) since there are already 3 for-loops (at 2 levels) in `toPaths` adding a level of for-loop will likely hurt the readability.